src/main/kotlin/io/github/rybalkinsd/kohttp/ext/ResponseExt.kt

```diff
@@ -76,3 +78,14 @@ data class EagerResponse(
  * @author sergey
  */
 data class Header(val name: String, val value: String)
+
+internal val stringMapper = ObjectMapper()
```

Comment:
In my mind not every client will use `asJson`, however `stringMapper` will be allocated in any client application.

I'd suggest to use a kotlin lazy initialization here for `stringMapper`